virtual-repeater.js

```diff
@@ -50,7 +42,77 @@ export const Repeats = Superclass => class extends Superclass {
     // Used to keep track of measures by index.
     this._indexToMeasure = {};
 
-    this._scheduleRender();
+    if (config) {
+      Object.assign(this, config);
+    }
+  }
+
+  // API
+
+  get container() {
```

Comment:
Seems like there's a few things not documented here. Worth filing an issue to keep track of them so we can decide whether to document them or make them internal.

Reply:
Yeah, these components need some documentation indeed. I wanted to decide first how these will be shipped, in order to focus efforts where is needed first.
Tracking issue https://github.com/PolymerLabs/virtual-list/issues/27